contracts/multihop/src/storage.rs

```diff
@@ -68,14 +68,14 @@ pub fn set_initialized(e: &Env) {
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct SimulateSwapResponse {
     pub ask_amount: i128,
-    pub total_commission_amount: i128,
+    pub commission_amounts: Vec<(String, i128)>,
     pub spread_amount: Vec<i128>,
 }
 
 #[contracttype]
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct SimulateReverseSwapResponse {
     pub offer_amount: i128,
-    pub total_commission_amount: i128,
+    pub commission_amounts: Vec<(String, i128)>,
```

Comment:
Add doc comments comments that those strings are respectively offer and ask asset denoms.